### PR TITLE
[helm] Fabconnect Support

### PR DESCRIPTION
**Pull request by hfuss**
* Templates `blockchain.fabric` config section of `firefly.core` with new Helm values
* Chart CI values broken into two tests: `it-eth` and `it-fab` which tests setting connector values for both Ethereum and Fabric

### CODE REVIEW

**Review by drewmarshburn — APPROVED** [review]

File: deploy/charts/firefly/templates/_helpers.tpl [hunk 1]
```diff
@@ -124,6 +124,23 @@ blockchain:
...
+  fabric:
+    fabconnect:
+      url: {{ tpl .Values.config.fabconnectUrl . }}
+      {{- if and .Values.config.fabconnectUsername .Values.config.fabconnectPassword }}
+      auth:
+        username: {{ .Values.config.fabconnectUsername | quote }}
+        password: {{ .Values.config.fabconnectPassword | quote }}
+      {{- end }}
+      retry:
+        enable: {{ .Values.config.fabconnectRetry }}
```
Comment (peterbroadhurst) [inline deploy/charts/firefly/templates/_helpers.tpl:139]: Think this is a great thing to add, but we should add it consistently for `ethconnectRetry` too

**Review by peterbroadhurst — APPROVED** [review]
One suggestion @hfuss to consider, with approval

File: deploy/charts/firefly/templates/_helpers.tpl (hunk 1 above)
Comment (hfuss) [reply deploy/charts/firefly/templates/_helpers.tpl:148]: done, thanks!

**Comment by hfuss** [comment]
Checks are failing and will need to be rerun due to ongoing DockerHub outage.

**Pull request merged**